src/App.js

```diff
@@ -9,6 +9,7 @@ import NotesListing from "./pages/notes-page/NotesListing";
 import RequiresAuth from "./utils/require-auth/RequiresAuth";
 import ArchiveListing from "./pages/notes-page/ArchiveListing";
```

Comment:
you can do a barrel import / export here, basically for all exports / imports. 

makes the code look cleaner